src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -194,9 +219,7 @@ private protected (Solution oldSolution, Solution newSolution) SetCurrentSolutio
         {
             var (updated, _) = SetCurrentSolution(
                 transformation,
-                (_, _) => changeKind,
-                projectId,
-                documentId,
+                (_, _) => (changeKind, projectId, documentId),
```

Comment:
this was just a simplifying transformation that made other changes/callers simpler.